.travis.yml

```diff
@@ -6,9 +6,8 @@ addons:
       - softhsm
 
 go:
-  - 1.5
-  - 1.6
-  - 1.7
+  - 1.11
+  - 1.12
```

Comment:
I think these need to be "1.12.x" and "1.11.x" in order to get the latest point release in each series. As you have it will result in getting exactly `1.11.0` and `1.12.0` as confirmed by [CI output for this repo](https://travis-ci.org/letsencrypt/pkcs11key/jobs/559620925):
```
$ go version
go version go1.12 linux/amd64
```

Versus [CI output](https://travis-ci.org/letsencrypt/ct-woodpecker/jobs/561844058) from a repo using `"1.12.x"`:
```
$ go version
go version go1.12.7 linux/amd64
```